airsonic-main/src/main/java/org/airsonic/player/service/sonos/SonosServiceRegistration.java

```diff
@@ -19,7 +19,6 @@
 
 package org.airsonic.player.service.sonos;
 
-import org.airsonic.player.dao.SonosLinkDao;
 import org.airsonic.player.util.Pair;
```

Comment:
I think this whole commit can be removed once you rebase to remove the original change.